lib/trino-filesystem-manager/src/test/java/io/trino/filesystem/manager/TestFileSystemConfig.java

```diff
@@ -35,7 +36,7 @@ public void testDefaults()
                 .setNativeGcsEnabled(false)
                 .setNativeLocalEnabled(false)
                 .setCacheEnabled(false)
-                .setTrackingEnabled(false));
+                .setTrackingEnabled(getenv("CONTINUOUS_INTEGRATION") != null));
```

Comment:
Nit: could save this in a field in the test to avoid the duplication (you can ignore this)